src/usr/local/www/guiconfig.inc

```diff
@@ -28,8 +28,22 @@
 if (!$nocsrf) {
 	function csrf_startup() {
 		csrf_conf('rewrite-js', '/csrf/csrf-magic.js');
-		$timeout_minutes = isset($config['system']['webgui']['session_timeout']) ? $config['system']['webgui']['session_timeout'] : 240;
-		csrf_conf('expires', $timeout_minutes * 60);
+
+//FIXME: 
+// This code doesn't seem to work as expected, because $config[] isn't accessible within csrf_startup() when it executes, so it mis-sets csrf timeout. 
+// Ref - https://redmine.pfsense.org/issues/6803 (1st note)
```

Comment:
That doesn't fix it. See discussion on Redmine 6803. The problem is that $config not only isn't declared via a "global" statement - it isn't available and hasn't been rread from file at that point. So the whole reading of config.xml + any decrypting, etc, is needed, or some workaround, not just adding a "global $config" statement. The above "fix" wasn't a fix.